packages/demo/frontend/src/envVars.ts

```diff
@@ -0,0 +1,20 @@
+import { z } from 'zod'
+
+const envVarSchema = z.object({
+  VITE_VERBS_API_URL: z
+    .string()
+    .url()
+    .default(
+      import.meta.env.MODE === 'production'
```

Comment:
🎨 `MODE` is provided by vite:
`pnpm run dev` sets `development`
`pnpm preview` sets `production`